nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -1503,6 +1508,12 @@
     dirs:
       - 'grammars'
 
+- module-name: 'libusb_package'
+  dlls:
+    - from_filenames:
+        prefixes:
+          - 'libusb-1.0'
```

Comment:
Maybe lets be less precise here and go with `libusb-` such that version changes there will not require to add to there in the future.

Reply:
get it